src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -104,6 +104,8 @@ public Task Get()
         {
             var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("name");
             var node = GetStringQueryString("node", false);
+            var encrypted = GetBoolValueQueryString("encrypted", required: false) ?? false;
```

Comment:
Remove this, we are ALWAYS encrypted if the db is encrypted, or we are never encrypted if the db is not encrypted.
In other words, you can only specify encryption during db creation, not when adding a node